CONTRIBUTING.md

```diff
@@ -81,3 +81,41 @@ See #284
 
 Push your branch to your `goreleaser` fork and open a pull request against the
 master branch.
+
+
+## Financial contributions
+
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/goreleaser).
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+
+## Credits
+
+
+### Contributors
+
+Thank you to all the people who have already contributed to goreleaser!
+<a href="graphs/contributors"><img src="https://opencollective.com/goreleaser/contributors.svg?width=890" /></a>
```

Comment:
this URL doesn't seem to work...

Reply:
bug submitted and actively being addressed at open collective. I'll check back here after status changes. Thanks!!!

Reply:
Thanks =D

Reply:
It works!

Reply:
neat, thankkks!